aspnetcore/blazor/hybrid/class-libraries.md

```diff
@@ -0,0 +1,514 @@
+---
+title: Share assets across web and native clients using a Razor class library (RCL)
+author: guardrex
+description: Learn how to share Razor components, C# code, and static assets across web and native clients using a Razor class library (RCL).
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/07/2022
+uid: blazor/hybrid/class-libraries
+---
+# Share assets across web and native clients using a Razor class library (RCL)
+
+Use a Razor class library (RCL) to share Razor components, C# code, and static assets across web and native client projects.
+
+This article builds on the general concepts found in the following articles:
+
+* <xref:blazor/components/class-libraries>
+* <xref:razor-pages/ui-class>
+
+The examples in this article share assets between a Blazor Server app and a .NET MAUI Blazor app in the same solution:
+
+* Although a Blazor Server app is used in examples and described by the text as the "web" client, the guidance applies equally to Blazor WebAssembly apps.
+* The example projects are in the same solution, but an RCL can supply shared assets to projects outside of the same solution.
+* The RCL example is added as a project to the solution, but any RCL can be published as a NuGet package. A NuGet package can supply shared assets to web and native client projects.
+* The order that the projects are created isn't important. However, projects that rely on an RCL for assets must create a project reference to the RCL *after* the RCL is created.
+* The following project names are used for the projects in this article, which become part of namespace and assembly references in the examples:
+  * RCL: `SharedLibrary`
+  * Blazor Hybrid (.NET MAUI Blazor): `BlazorNative`
```

Comment:
We've used "Blazor Native" in the past to refer to using Blazor and Razor syntax to build native UI, so there's a small potential for confusion here. Could we call the Blazor Hybrid app "MauiBlazorApp" or "BlazorHybridApp" instead?